cpp/ps0002/the third task.cpp

```diff
@@ -1,30 +1,27 @@
-﻿
-
 #include <iostream>
 
 int main()
 {
-    std::cout << "Hello World!\n";
     //عدد سكان عماره
-    short architecture = 30;
-   // عدد البدايل لدوا في الصيدليه 
-    int dragalternatives = 200;
-  //  افراد تيم فشركه برمجه
-    unsigned embloyees = 50;
-   // سعر واحده من مخبوزات الفرن
-     float bread = 5;
+    short architecture = 30; // 0.75/0.75
+    // عدد البدايل لدوا في الصيدليه 
+    short dragalternatives = 200; // 0.5/0.75
+    //  افراد تيم فشركه برمجه
+    short embloyees = 50; // 0.75/0.75
+    // سعر واحده من مخبوزات الفرن
+    float bread = 5; // 0.75/0.75
     //الحرف الاول من اسم شخص 
-    char name ='Y';
-   // عدد المواليد
-    long birth = 1425114339;
-   // ناتج قسمه 12 على 7
-    float number=1.714;
+    char name ='Y'; // 0.75/0.75
+    // عدد المواليد
+    int birth = 1425114; // 0.25/0.75
+    // ناتج قسمه 12 على 7
+    double number=1.714; // 0.5/0.75
     //فاتوره الكهرباء 
-    float reseat = 12650;
+    float reseat = 12650; // 0.75/0.75
     //عمر رجل لديه 50 سنه من عمره بالثانيه 
-    unsigned age = 4320000;
-
+    unsigned age = 4320000; // 0.75/0.75
+    // total 5.75/6.75
 
     return 0;
 }
-
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code